connect-inject/endpoints_controller.go

```diff
@@ -156,30 +162,42 @@ func (r *EndpointsController) Reconcile(ctx context.Context, req ctrl.Request) (
 					// Note: the order of how we register services is important,
 					// and the connect-proxy service should come after the "main" service
 					// because its alias health check depends on the main service existing.
-					r.Log.Info("registering service with Consul", "name", serviceRegistration.Name)
-					err = client.Agent().ServiceRegister(serviceRegistration)
-					if err != nil {
-						r.Log.Error(err, "failed to register service", "name", serviceRegistration.Name)
-						return ctrl.Result{}, err
-					}
+					if managedByEndpointsController {
+						//fmt.Println("here")
```

Comment:
```suggestion
```

Reply:
Thank you!!! Sorry I missed some cleanups!!